README.md

```diff
@@ -127,10 +127,10 @@ When you open the Command Palette in VSCode (⇧⌘P or Ctrl+Shift+P), this plug
 ## How to develop
 
 1. Fork this repo, and clone your fork locally.
-2. Run `npm install` right under project root.
+2. Run `npm install` right under project root and `/client` and `/server`.
```

Comment:
You only need to run `npm install` under project root, it will work thanks to this `postinstall` script :
https://github.com/standard/vscode-standard/blob/28d1faefe97af78257be65258efaacee15f09cd6/package.json#L213

Reply:
Updated

I have `--ignore-scripts` disabled